html/admin/banpanel.js

```diff
@@ -8,3 +8,12 @@ function toggle_other_checkboxes(source, copycats_str, our_index_str) {
         document.getElementById(source.id.slice(0, -1) + i).checked = source.checked;
     }
 }
+
+function header_click_all_checkboxes(source) {
+	var checkboxes = document.getElementsByClassName(source.name);
+	for(var i = 0, n = checkboxes.length; i < n; i++) {
+		if(checkboxes[i].checked != source.checked && checkboxes[i] != source){
```

Comment:
I dont want to open VSC
```suggestion
		if(checkboxes[i].checked != source.checked && checkboxes[i] != source) {
```